src/content/en/fundamentals/web-app-manifest/index.md

```diff
@@ -102,20 +102,24 @@ automatically scale the icon for the device. If you'd prefer to scale your
 own icons and adjust them for pixel-perfection, provide icons in increments
 of 48dp.
 
+Note: "sizes" is plural because it's a string of space-separate image
```

Comment:
Not sure how relevant this section is, icons should be png, and I'd rather not confuse folks with ico files.